hadoop-hdds/docs/content/interface/OzoneFS.zh.md

```diff
@@ -58,7 +54,7 @@ ozone sh bucket create /volume/bucket
 你还需要将 ozone-filesystem.jar 文件加入 classpath：
 
 {{< highlight bash >}}
-export HADOOP_CLASSPATH=/opt/ozone/share/ozonefs/lib/hadoop-ozone-filesystem-lib-current*.jar:$HADOOP_CLASSPATH
+export HADOOP_CLASSPATH=/opt/ozone/share/ozonefs/lib/hadoop-ozone-filesystem-hadoop3-*.jar:$HADOOP_CLASSPATH
 {{< /highlight >}}
 
```

Comment:
Please insert between 59-60 the following in Chinese for the new sentence added 
“(Note: with Hadoop 2.x, use the `hadoop-ozone-filesystem-hadoop3-*.jar`)”

(注意：当使用Hadoop 2.x时，应该在classpath上添加`hadoop-ozone-filesystem-hadoop2-*.jar`）

Reply:
> Please insert between 59-60 the following in Chinese for the new sentence added

Thanks A LOT. 

BTW, what is at �? I installed Chinese fonts, but I couldn't see it. I am not sure if I it's because my copy-paste or Github removed from something from there...

Reply:
I think that's either typos or github encoding issue when I switch from Chinese to English on the comment. You can ignore and remove them.

Reply:
done